_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+
+C. Department of Materials Science and Engineering, \\
+Northwestern University, \\
+Evanston, IL 60208 USA
+
+D. Fuel Modeling and Simulation Department, \\
+Idaho National Laboratory, \\
+Idaho Falls, ID 83415 USA
+
+E. Materials Science and Engineering Department, \\
+University of Michigan, \\
+Ann Arbor, MI 48109 USA
+
+F. Material Measurement Laboratory Office, \\
+Material Measurement Laboratory, \\
+National Institute of Standards and Technology,\\
+Gaithersburg, MD 20899 USA
+
+G. Department of Materials Science and Engineering, \\
+University of Florida, \\
+Gainesville, FL 32611
+
+\section*{Abstract}
+
+An online portal provides a valuable space for scientific communities
+to summarize a shared challenge, collect attempts at a solution, and
+present a quantitative comparison of past attempts in a compelling
+way. An exemplar of such a portal is
+$\mu$MAG~\cite{mumag}. The reusable PFHub
+framework leverages existing online services to build a static portal
+website that is considerably easier to deploy and maintain without
+sacrificing content or scope. The first deployment of the PFHub
+framework supports phase-field practitioners and code developers
+participating in an effort to improve quality assurance for
+phase-field codes.
+
+%% ~\footnote{Certain commercial equipment,
+%%   instruments, or materials (or suppliers, or software, ...) are
+%%   identified in this paper to foster understanding. Such
+%%   identification does not imply recommendation or endorsement by the
+%%   National Institute of Standards and Technology, nor does it imply
+%%   that the materials or equipment identified are necessarily the best
+%%   available for the purpose.\label{disclaimer}}
+
+%% ~\textsuperscript{\ref{disclaimer}}
+
+\section*{Keywords}
+
+phase-field; materials-science; jekyll-website; reproducible-science
+
+\section*{Introduction}
+
+The phase-field method (PFM) describes material interfaces at the
+mesoscopic scale between atomic scale models and macroscale models.
+The PFM is well established and there are an assortment of code
+frameworks (\emph{e.g.}, MOOSE~\cite{moose},
+PRISMS-PF~\cite{prisms-pf}, FiPy~\cite{fipy},
+MMSP~\cite{mmsp}) available for
+solving the wide variety of mesoscale phenomena (\emph{e.g.},
+dendritic growth, spinodal decomposition, grain growth). However,
+phase-field research groups often develop codes in isolation and do
+not support or distribute the code bases to the wider community. PFHub
+is a community effort spearheaded by the Center for Hierarchical
+Materials Design at Northwestern University and the National Institute
+of Standards and Technology to support the development of phase-field
+codes. In particular, the PFHub effort focuses on improving
+cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark
+problems~\cite{bm1, bm2} along with a web framework for uploading and
+comparing benchmark results from different codes.
```

Comment:
The introduction does not do a good job of introducing what the paper is actually about. The content of the paper has nothing to do with the phase field method. Rather, it is focused on summarizing the details of the architecture used to facilitate the sharing of results. This focus is clear in the abstract, but this introduction talks about why PFHub is needed by the phase field community, not why a set of tools like this would be useful for any computational community that wants a portal for sharing results. The intro needs to be rewritten to better reflect the focus of the paper.